modules/core/src/lib/composite-layer.js

```diff
@@ -191,7 +191,10 @@ export default class CompositeLayer extends Layer {
     for (const extension of extensions) {
       const passThroughProps = extension.getSubLayerProps.call(this, extension);
       Object.assign(newProps, passThroughProps, {
-        updateTriggers: Object.assign(newProps.updateTriggers, passThroughProps.updateTriggers)
+        updateTriggers: Object.assign(
```

Comment:
Without this change, new unit tests added will crash, `newProps.updateTriggers` was undefined or null.

Reply:
How is that possible with line 179?

Reply:
In my testcase, argument to `getSubLayerProps()` function, `sublayerProps` is undefined. Also it will only happen, if `extensions` is not an empty array, so I guess thats not being tested with exsisting unit tests of `CompositeLayer`.  One fix is, set a default argument (`getSubLayerProps(sublayerProps = {}) {...`)